docker-compose-development-database.yml

```diff
@@ -0,0 +1,15 @@
+version: "3"
```

Comment:
Can we name these `docker-compose.yml` and `docker-compose.test.yml`, just to be consistent with what we have in the UI?

Reply:
Sure thing, have rebased in